pkg/patterns/declarative/pkg/applier/exec_test.go

```diff
@@ -144,6 +144,18 @@ metadata:
 }
 
 func TestKubectlApplier(t *testing.T) {
+	kubectlPath, err := exec.LookPath("kubectl")
```

Comment:
would this prefer ./bin over standard paths ?

Reply:
Only if `./bin` is set in the PATH, which we do here :-)

The goal here is primarily to give a nice error if kubectl isn't on the path, but also to print the path to kubectl and the version we found, so that if something weird happens that looks like a kubectl version issue we can figure out what the problem is from the logs!